client-js/evomaster-client-js/src/instrumentation/heuristic/HeuristicsForBooleans.ts

```diff
@@ -332,4 +333,31 @@ export default class HeuristicsForBooleans {
         return res;
     }
 
+    public static handleTernary(f: () => any, fileName: string, line: number, index: number) {
+
+        // TODO: Man what is this for? do I need this?
+        HeuristicsForBooleans.lastEvaluation = null;
```

Comment:
it was explained in the paper. but i guess we should put it in a (private) function (e.g., `resetLastEvaluation`), with some comments like: `Make sure that nested evaluations of && and || do not use unrelated previous computation`.

I am not sure if it is really needed here, but can leave it for now